src/operator/tensor/sparse_retain.cc

```diff
@@ -37,14 +41,14 @@ Example::
   shape = (4, 2)
   rsp_in = row_sparse(data, indices)
   to_retain = [0, 3]
-  rsp_out = sparse_retain(rsp_in, to_retain)
+  rsp_out = _sparse_retain(rsp_in, to_retain)
```

Comment:
use retain instead of _sparse_retain in the doc? Since user wont' see `_sparse_`